packages/langium/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log of `langium`
 
+## 3.3.0 (Nov. 2024)
+
```

Comment:
Let's add a link to https://gitlab.eclipse.org/eclipsefdn/emo-team/emo/-/issues/810 and announce the graduation of our project :-)

Reply:
Good idea!